client-src/elements/chromedash-feature-detail.js

```diff
@@ -2,6 +2,7 @@ import {LitElement, css, html, nothing} from 'lit';
 import {getStageValue, renderHTMLIf} from './utils';
 import {enhanceUrl} from './feature-link';
 import {openAddStageDialog} from './chromedash-add-stage-dialog';
+// import {openPrereqsDialog} from './chromedash-ot-create-prereqs-dialog';
```

Comment:
Don't you need this import to not be commented out?

Reply:
OK, I missed the fact that the code below was in a block comment because GitHub did not display that part or take it into account for syntax coloring.